src/utiles/appx.ts

```diff
@@ -91,5 +101,6 @@ export async function getPurchases(email: string) {
       }
     }
   }
+  Cache.getInstance().set('courses', [email], responses, 60 * 60 * 24);
```

Comment:
only this line and the cache get for this is needed
No need for any LOCAL_CMS_PROVIDER logic that can remain uncached